hack/ifreload/Dockerfile.tools

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi
+RUN dnf -y update \
+  && dnf -y install libnl3-devel make gcc \
+  && dnf clean all
```

Comment:
for just a build tool do we really need these update and clean steps? it would be faster with just the install step

Reply:
subsequent builds will cache layers